fairlearn/datasets/_utils.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+# State Code based on 2010 Census definitions
```

Comment:
"... for ACS datasets"
Perhaps also add a link to the folktables repo.

Reply:
Clarified the description!